argo-workflows/sync-nb-server-to-ironic/workflowtemplates/sync-server-to-ironic.yaml

```
@@ -11,7 +11,12 @@ spec:
       - name: interface_update_event
         value: "{}"
   templates:
-    - name: sync-server
+    - name: main
```

```suggestion
    - name: sync-server
```

The parent workflow has a ref to `sync-server` so it will break that. We need to ultimately add some tests to check references.

---

oops, I missed that when I was changing things. Fixed it now